index.ts

```diff
@@ -67,14 +67,17 @@ const plugin: Plugin<[(RemarkEmojiOptions | null | undefined)?], Root> = options
     function replaceEmoticon(match: string): string | false | Text {
         // find emoji by shortcode - full match or with-out last char as it could be from text e.g. :-),
         const iconFull = emoticon.find(e => e.emoticons.includes(match)); // full match
-        const iconPart = emoticon.find(e => e.emoticons.includes(match.slice(0, -1))); // second search pattern
-        const icon = iconFull || iconPart;
+        const iconPartEnd = emoticon.find(e => e.emoticons.includes(match.slice(0, -1))); // second search pattern
```

Comment:
Are these names swapped? End seems to be the front of the match?

Reply:
That was the exact question I came up with lol.  I was thinking the trimmed part, but I think you're right and I'll swap them